src/components/Task/Task.js

```diff
@@ -1,3 +1,16 @@
-export const Task = ({ value }) => {
-  return <li>{value}</li>;
+import { Button } from "../Button/Button";
+import { useTasksStore } from "../../hooks/UseTasksStore";
+
+export const Task = (item) => {
+  const { removeTask, toggleDoneTask } = useTasksStore(item.id, item);
```

Comment:
Do not pass the whole `item`. Only pass an `id` and filter inside of the hook. This is important because on unit-level testing leads to inconsistency, e.g. if an id is passed from another item. Yes, that shouldn't normally happen, but better not to duplicate data.